license-sale-arb-sepolia/src/play-fi-license-sale.ts

```diff
@@ -8,80 +8,81 @@ import {
   TeamSaleStatusSet, TierSet
 } from "../generated/PlayFiLicenseSale/PlayFiLicenseSale"
 import {Account, Claim, Phase, Referral, Stat, Tier} from "../generated/schema"
-import {BigInt, Bytes} from "@graphprotocol/graph-ts";
+import {Address, BigInt, Bytes, ByteArray, log} from "@graphprotocol/graph-ts";
 
 export function handleContractInitialized(event: ContractInitialized): void {
   //Initialize stats
-  let stat = new Stat(Bytes.fromI32(0));
+  let stat = new Stat(BigInt.zero().toString());
   stat.totalClaims = BigInt.zero();
   stat.totalProceeds = BigInt.zero();
   stat.save()
 
   //Set team phase paused
-  let teamPhase = new Phase(Bytes.fromI32(0));
+  let teamPhase = new Phase(BigInt.zero().toString());
```

Comment:
q: would it make more sense to set an id number to another value than the init value?

Reply:
There is no default value in subgraph. It either exists or it doesn't. It's not like in the EVM where the default values actually exist.